src/main/java/com/opendxl/databus/consumer/DatabusPushConsumerListener.java

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------------*
+ * Copyright (c) 2019 McAfee, LLC - All Rights Reserved.                     *
+ *---------------------------------------------------------------------------*/
+
+package com.opendxl.databus.consumer;
+
+/**
+ * This is the interface that SDK Databus client has to implement to receive and process records
+ * when consume them by using a {@link DatabusPushConsumer} instance. {@link DatabusPushConsumer#pushAsync()}
+ * main loop read messages from an already-subscribed topic and send them
+ * to {@link DatabusPushConsumerListener#onConsume(ConsumerRecords)} method.
+ * It will process messages and will return a {@link DatabusPushConsumerListenerResponse}.
+ * According to it, DatabusPushConsumer will act accordingly.
+ * <p>
```

Comment:
Cosmetic:  are these lines misplaced? I see CONTINUE_AND_COMMIT and other enum values shown below.